org.eclipse.buildship.core.test/src/main/groovy/org/eclipse/buildship/core/workspace/internal/ImportingWtpProjects.groovy

```diff
@@ -23,6 +24,34 @@ class ImportingWtpProjects extends ProjectSynchronizationSpecification {
     private static final String NON_DEPLOYED = "org.eclipse.jst.component.nondependency"
     private static final String DEPLOYED = "org.eclipse.jst.component.dependency"
     private static final String WTP_COMPONENT_NATURE = "org.eclipse.wst.common.modulecore.ModuleCoreNature"
+	
+	def "Do not check mixed deployment paths without WTP installed"() {
```

Comment:
❌ The test fails in my local eclipse as I have WTP installed. I'd fix it the following way:
- Introduce the `isWtpInstalled()` method on the `WorkspaceOperations` interface.
- Adjust `WorkspaceOperationsDelegate` to return a configurable result for `isWtpInstalled`. 
- Duplicate this test to cover both the use cases when WTP is and is not installed.